Extensions/blacklist.js

```diff
@@ -1174,6 +1178,287 @@ XKit.extensions.blacklist = new Object({
+					.map(word => word.replace('#', '').trim())
+					.filter(word => !currentFilteredTags.includes(word));
+
+				const newContentWords = blacklistItemData
+					.filter(({ contentCheckbox }) => contentCheckbox.checked)
+					.flatMap(({ textInput }) => textInput.value.split(','))
+					.map(word => word.replace('#', '').trim())
+					.filter(word => !currentFilteredContent.includes(word));
+
+				if (newTagWords.length || newContentWords.length) {
+					Promise.all([
+						newTagWords.length &&
+							apiFetch('/v2/user/filtered_tags', {
+								method: 'POST',
+								body: { filtered_tags: newTagWords },
+							}),
+						newContentWords.length &&
+							apiFetch('/v2/user/filtered_content', {
+								method: 'POST',
+								body: { filtered_content: newContentWords },
+							}),
+					]).then(() => XKit.window.show(
+						'Success',
+						`
+							${newTagWords.length ? `<div>Added filtered tags: ${newTagWords.map(word => `#${word}`).join(', ')}</div>` : ''}
+							${newContentWords.length ? `<div>Added filtered content: ${newContentWords.join(', ')}</div>` : ''}
+							<div>
+								You can see your native filtered content in your
+								<a href="https://www.tumblr.com/settings/account" target="_blank">
+								Tumblr account settings</a>.
+							</div>
+						`,
+						'info',
+						'<div id="xkit-close-message" class="xkit-button">Close</div>',
+					)).catch(showNativeExportError);
+
+				}
+			};
+
+			const createRow = (data) =>
+				dom('tr', null, null, data.map((contents) => dom('td', null, null, [contents])));
+
+			const rows = blacklistItemData.map(({ name, textInput, tagCheckbox, contentCheckbox }) =>
+				createRow([name, textInput, tagCheckbox, contentCheckbox])
+			);
+
+			const containerId = 'blacklist-native-container';
+			const tableId = 'blacklist-native-export-table';
+			const tableBodyId = 'blacklist-native-export-table-body';
+			const doExportId = 'blacklist-native-export-do-export';
+			const selectAllId = 'blacklist-native-export-select-all';
+			const selectNoneId = 'blacklist-native-export-select-none';
+
+			XKit.window.show(
+				'Tumblr Native Filtering Export',
+				`
+					<div id=${containerId}>
+						<div>
+							Tumblr's native filtering has two categories of words: filtered tags and
+							filtered post content.
+						</div>
+						<div>
+							At the time this message was last updated:
+						</div>
+						<ul>
+							<li>
+								Filtering a word or phrase as a tag will hide any post with that exact
+								tag (no wildcards) and reblog chains whose original root posts contain
+								that exact tag.
+							</li>
+							<li>
+								Filtering a word or phrase as post content will hide any post with the
+								specified word or phrase anywhere in the post text or in any usernames,
+								including in the middle of a word (filtering "ash" will hide posts with
+								"dashboard" or "fashion"). It will not search the post tags.
```

Comment:
Might be nice to add an example for usernames as well:

```suggestion
								Filtering a word or phrase as post content will hide any post with the
								specified word or phrase anywhere in the post text or in any usernames,
								including in the middle of a word (filtering "ash" will hide posts with
								"dashboard" or "fashion", or with a comment by a user named
								"ash-ketchum"). It will not search the post tags.
```